src/biotite/sequence/io/fasta/convert.py

```diff
@@ -28,6 +28,9 @@ def get_sequence(fasta_file, header=None):
     ----------
     fasta_file : FastaFile
         The :class:`FastaFile` to be accessed.
+    seq_type : Class, optional
+        The :class:`Sequence` subclass contained in the file. If not 
+        set, it is automatically detected.
```

Comment:
Could you also mention that only nucleotide and protein sequences can be automatically detected?